README.md

```diff
@@ -9,3 +9,50 @@
 [releases]: https://github.com/vshn/appcat-sli-exporter/releases
 
 The ApppCat SLI Exporter is a Prometheus that automatically detects AppCat services and measures uptime SLIs.
+
+
+## Building
+
+See `make help` for a list of build targets.
+
+* `make build`: Build binary for linux/amd64
+* `make build -e GOOS=darwin -e GOARCH=arm64`: Build binary for macos/arm64
+* `make docker-build`: Build Docker image for local environment
+
+## Metrics
+
+The exporter exposes a histogram `appcat_probes_seconds` with four labels
+
+ * `service`, the service type that was probed (e.g. `VSHNPostgreSQL`)
+ * `namespace`, the namespace of the claim that was monitored
+ * `name`, the name of the claim that was monitored
+ * `reason`, if the probe was successful. Can either be `success`, `fail-timeout`, or `fail-unkown`
+
+## Architecture
+
+```
+.
+├── config // Kustomize files to deploy the exporter
```

Comment:
Any reason why not add the manifests in our component-appcat instead of using kustomize?

Reply:
It will be added. But we'll use kustomize to render them in the component-appact (You'll see what I mean in that PR :smiley: )